helm/loki/templates/minio/minio-ciliumnetworkpolicy.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.loki.minio.enabled -}}
+{{- if .Values.ciliumNetworkPolicy.enabled -}}
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: minio-cilium-netpol
+  namespace: kube-system
+spec:
+  egress:
+  - toEntities:
+    - kube-apiserver
+  endpointSelector:
+    matchLabels:
+      app: minio
+  ingress:
+  - toPorts:
+    - ports:
+      - port: "9000"
```

Comment:
Is that not part of the values ?

Reply:
Also it would be nice upstream ;)

Reply:
using networkPolicy.flavor=cilium :)

Reply:
yeah adding this upstream could be a good idea :) 

> Is that not part of the values ?

Unfortunately the minio's port can't be set from the values :/